lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -606,8 +607,7 @@ def getvm(self, name=None, uuid=None, folder=None, name_match=None, cache=False)
                 paths = [x for x in paths if x.endswith(self.params['folder'])]
                 if len(paths) > 1:
                     self.module.fail_json(
-                        msg='%s matches more than one folder. Please use the absolute path starting with /vm/' %
-                            self.params['folder'])
+                        msg='%(folder)s matches more than one folder. Please use the absolute path starting with /vm/' % self.params)
```

Comment:
IMHO everybody should aim for PEP8 compliance unless it has some very negative impact like creating new variables only to be able to fit into the 80 chars limit.